scripts/migrations/index.mjs

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env node
+
+import yargs from 'yargs'
+import { hideBin } from 'yargs/helpers'
+
+import { config } from 'dotenv-safe'
+config()
+
+import { spawnSync } from 'child_process'
+
+const {
+  DATO_FULL_ACCESS_API_TOKEN_RWS_VIEWER,
+  DATO_FULL_ACCESS_API_TOKEN_DELTARES_VIEWER,
```

Comment:
Let's rename this to `DATO_FULL_ACCESS_API_TOKEN_OPENEARTH_VIEWER`

Reply:
Which one should be renamed, `DATO_FULL_ACCESS_API_TOKEN_RWS_VIEWER` or `DATO_FULL_ACCESS_API_TOKEN_DELTARES_VIEWER`?